### ISSUE TITLE
Warning when FlatLaf is used on the Mac

### ISSUE THREAD

**Issue by hshafie** [JFormDesigner/FlatLaf#64]
Just to inform you.

When I select FlatLaf 0.26  for my application on a Mac, latest macOS Catalina, latest Java 13.0.2, _it works and looks fine_, **but** I get this warning printed on the Terminal I've used to start my application:

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.formdev.flatlaf.FlatLaf (file:/Users/hussein/Library/Application%20Support/XMLmind/XMLEditor9/addon/flatlaf/flatlaf.jar) to constructor com.apple.laf.AquaLookAndFeel()
WARNING: Please consider reporting this to the maintainers of com.formdev.flatlaf.FlatLaf
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

After looking at your code, this warning _may_ be caused by this excerpt from `com/formdev/flatlaf/FlatLaf.java`:

```
	@Override
	public void initialize() {
		getBase().initialize();

		super.initialize();
...

	/**
	 * Get/create base LaF. This is used to grab base UI defaults from different LaFs.
	 * E.g. on Mac from system dependent LaF, otherwise from Metal LaF.
	 */
	private BasicLookAndFeel getBase() {
		if( base == null ) {
			if( SystemInfo.IS_MAC ) {
				// use Mac Aqua LaF as base
				String aquaLafClassName = "com.apple.laf.AquaLookAndFeel";
				try {
					base = (BasicLookAndFeel) Class.forName( aquaLafClassName ).newInstance();
				} catch( Exception ex ) {
					LOG.log( Level.SEVERE, "FlatLaf: Failed to initialize base look and feel '" + aquaLafClassName + "'.", ex );
					throw new IllegalStateException();
				}
			} else
				base = new MetalLookAndFeel();
		}
		return base;
	}
```

**Comment by DevCharly** [comment]
Many thanks for reporting. This is fixed in just released 0.27

**Comment by hshafie** [comment]
Thank you for reacting so quickly! Downloaded 0.27 and using it in my app (https://www.xmlmind.com/xmleditor/). Works great.

**Comment by DevCharly** [comment]
@hshafie that's great! Please let me know when you release a version that uses FlatLaf.

**Comment by j-dimension** [comment]
> Please let me know when you release a version that uses FlatLaf.

How about maintaining a list of projects using FlatLaf. Could be a win-win :-)

**Comment by DevCharly** [comment]
The list already exists 😉 
https://github.com/JFormDesigner/FlatLaf#projects-using-flatlaf

**Comment by j-dimension** [comment]
Wow, thumbs up! (and not just for the list)

**Comment by hshafie** [comment]
> @hshafie that's great! Please let me know when you release a version that uses FlatLaf.

Hello,

XMLmind XML Editor v9.3, https://www.xmlmind.com/xmleditor/, to be released in the next few weeks, will propose a new add-on which will make it easy for a user to try and then probably adopt the FlatLaf Look&Feel.

This add-on will be introduced as follows:

> New add-on called "FlatLaf Look and Feel" containing a clean, simple and elegant “Look & Feel” for XMLmind XML Editor. Many thanks to [FormDev](https://www.formdev.com/flatlaf/) Software for having implemented such a nice, fast and reliable Look & Feel for Java Swing desktop applications!
> 
> This add-on is not installed by default. Feel free to download and install it using "Options|Install Add-ons". Once this is done, please select "FlatLight" (light theme) or "FlatDark" (dark theme) from [Options|Preferences, General section, Style combobox](https://www.xmlmind.com/xmleditor/_distrib/doc/help/generalOptions.html), then restart XMLmind XML Editor as instructed.
> 
> This add-on is supported on all platforms. It is especially nice to have on Linux where the default “cross-platform”, Look & Feel (called "Metal") looks rather outdated. It may also interest Windows and Mac users who want a dark theme or simply prefer how FlatLaf looks compared to the default, “system”, Look & Feel.

Feel free to add XMLmind XML Editor to your list of projects using FlatLaf any time you want.

Many thanks for your hard work. Much appreciated.

Cheers,
